src/hooks/httpUtils.test.js

```diff
@@ -0,0 +1,113 @@
+import {
+  extractBodyFromEmitSocketEvent,
+  extractBodyFromWriteFunc,
+  isValidHttpRequestBody,
+} from './httpUtils';
+
+describe('httpUtils', () => {
+  test('extractBodyFromSocketEvent -> outputData flow', () => {
+    const emitArg = {
+      _httpMessage: {
+        _hasBody: true,
+        outputData: [{ data: 'HTTP BODY1\nHTTP BODY2' }],
+      },
+    };
+
+    const result = extractBodyFromEmitSocketEvent(emitArg);
+
+    expect(result).toEqual('HTTP BODY2');
```

Comment:
Why do we extract only the first line?

Reply:
The last line,  see the previous comment